Gopkg.toml

```diff
@@ -11,51 +11,34 @@ required = [
 
 [[constraint]]
   name = "github.com/container-storage-interface/spec"
-  version = "1.0.0-rc2"
+  version = "=1.0.0" # Set this to exactly 1.0.0 until we can update test code to handle 1.1.0
 
-[[override]]
-  name = "github.com/golang/protobuf"
-  version = "v1.1.0"
-
-[[constraint]]
-  branch = "master"
-  name = "github.com/golang/glog"
-
-[[constraint]]
-  name = "github.com/kubernetes-csi/csi-test"
-  version = "1.0.0-rc2"
-
-[[constraint]]
-  name = "google.golang.org/grpc"
-  version = "1.9.2"
 
+# Branches are used for k8s.io constraints because at the
+# moment we need latest 1.14 APIs for external-provisioner.
+# This can be set to an official 1.14 tag once 1.14 is released.
+# The dependency on external-provisioner should be removed with #60.
+#
+# Once snapshotter depends on only stable K8s APIs, then all constraints should be removable.
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.0-beta.1"
+  branch = "release-1.14"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.0-beta.1"
+  branch = "release-1.14"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.0-beta.1"
-
-[[override]]
-  name = "github.com/json-iterator/go"
-  version = "1.1.4"
+  branch = "release-11.0"
```

Comment:
Is this the latest stable branch that supports k8s 1.14?

Reply:
yes, the 10.0 release supports k8s 1.13, so the 11.0 is for 1.14.